django/views/decorators/http.py

```diff
@@ -58,20 +58,20 @@ def condition(etag_func=None, last_modified_func=None):
 
     The parameters are callables to compute the ETag and last modified time for
     the requested resource, respectively. The callables are passed the same
-    parameters as the view itself. The Etag function should return a string (or
+    parameters as the view itself. The ETag function should return a string (or
```

Comment:
It would be appropriate to make the renames in a separate commit unless this is the only one to be fixed in which case it's probably not worth separating it.